### DISCUSSION TITLE
code block CSS 新增語言標籤

### DISCUSSION THREAD

**Discussion by FizzyElt** [FizzyElt/functional-programming#59] [Ideas]
可以利用加 attribute 的方式在 code block 外框顯示該區塊使用什麼語言

**Comment by FizzyElt** [comment]
是否有辦法在 latex 語法上塞 html attribute

**Reply by dannypsnl** [reply]
應該是需要問 Stering

**Reply by dannypsnl** [reply]
email list: https://lists.sr.ht/~jonsterling/forester-discuss

**Comment by dannypsnl** [comment]
這個在 email list 有一些討論了，但短期應該不會有進展，我先關掉這個囉

**Reply by FizzyElt** [reply]
好的
